### PR TITLE
JsonAdaptedPerson class: Change `tagged` to `tags`

### LINKED ISSUE #147: Coding standard violation on Storage component
In the file src/main/java/seedu/address/storage/JsonAdaptedPerson.java on the master directory, I found out that the variable `tagged` is not in plural even though it represents a collection of objects, hence violating CS2103T coding standard.

Below shows lines 33-47 of the code for easy reference.

```java
    /**
     * Constructs a {@code JsonAdaptedPerson} with the given person details.
     */
    @JsonCreator
    public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
            @JsonProperty("email") String email, @JsonProperty("address") String address,
            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
        this.name = name;
        this.phone = phone;
        this.email = email;
        this.address = address;
        if (tagged != null) {
            this.tagged.addAll(tagged);
        }
    }
```

Here is the relevant reference in the SE-EDU Java coding standard.
image

### PR DESCRIPTION
**Pull request by Py0000**
Fixes #147 

Change the variable tagged to tags in the affected class (src/main/java/seedu/address/storage/JsonAdaptedPerson.java).
Re-opened as a new PR.

### CODE REVIEW

**Comment by Py0000** [comment]
Hi Prof @damithc, I have closed the previous PR #148 for this issue (#147) and re-opened it as a new PR over here as I might have accidentally screwed up my own local repository. Sorry for the inconvenience regarding this.

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 1]
```diff
@@ -28,21 +28,21 @@ class JsonAdaptedPerson {
...
+    private final List<JsonAdaptedTag> tags = new ArrayList<>();
 
     /**
      * Constructs a {@code JsonAdaptedPerson} with the given person details.
      */
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
             @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+            @JsonProperty("tagged") List<JsonAdaptedTag> tags) {
```
Comment (damithc) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java:39]: Should we change both instances of `tagged` in this line?

**Review by damithc — COMMENTED** [review]
Just a question on something I noticed in passing.

**Comment by damithc** [comment]
Also, we need a full commit message as per our usual practice.

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java (hunk 1 above)
Comment (Py0000) [reply src/main/java/seedu/address/storage/JsonAdaptedPerson.java:39]: Hi Prof, I have tried to change both instances of `tagged` in this line when i was fixing this bug. But changing the `@JsonProperty` tag will affect other files as well, as `tagged` is currently being used in those files.

**Comment by Py0000** [comment]
> Also, we need a full commit message as per our usual practice.

Sorry about that Prof. I will go and figure out how to edit the commit message.

**Comment by Py0000** [comment]
I have updated the commit message.

**Comment by damithc** [comment]
```java
@JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
```

I think it is best if we modify both occurrences of `tagged`, if possible (or else we introduce an inconsistency which can be argued as even worse than the naming anomaly being fixed). So, I would like to explore that option before merging this PR.

**Comment by damithc** [comment]
Fixed by #169 that builds on this PR.
Thanks again for this PR @Py0000

**Pull request closed without merging**